### PR TITLE
Fixes:

### PR DESCRIPTION
**Pull request by Quix0r**
- timeout of 1 second was to short for the default directory at https://dir.friendica.social
- Used \Some\Foo::class instead of '\Some\Foo' which type-safe and easier to find by your editor/IDE

Signed-off-by: Roland Häder <roland@mxchange.org>

### CODE REVIEW

File: src/classes/Pollers/Server.php [hunk 1]
```diff
@@ -245,7 +245,7 @@ private function getProbeResult(string $base_url): array
 		$handle = curl_init();
 		$options = array(
 			//Timeouts
-			CURLOPT_TIMEOUT => max($this->settings['probe_timeout'], 1), //Minimum of 1 second timeout.
+			CURLOPT_TIMEOUT => max($this->settings['probe_timeout'], 3), //Minimum of 3 second timeout.
```
Comment (MrPetovan) [inline src/classes/Pollers/Server.php]: 1 second is the absolute minimum but you can increase the timeout itself in your local config file. Please remove this change from your PR.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/classes/Pollers/Server.php (hunk 1 above)
Comment (AndyHee) [reply src/classes/Pollers/Server.php]: Thanks for the hint @MrPetovan !

Could you tell us how this configuration looks like in practice?

Comment (MrPetovan) [reply src/classes/Pollers/Server.php]: Sure, here's a sample `config/local.json` file with the setting updated:
```json
{
    "database": {
        "driver": "mysql",
        "hostname": "localhost",
        "database": "friendica-directory",
        "username": "friendica-directory",
        "password": "🤡"
    },
    "poller": {
        "server_poll_delay": 28800,
        "profile_poll_delay": 36000,
        "probe_timeout": 10
    },
    "logger": {
        "level": 250
    }
}
```

Comment (AndyHee) [reply src/classes/Pollers/Server.php]: Great! Thanks very much, @MrPetovan

**Comment by Quix0r** [comment]
I forced-pushed the revert of the timeouts. Now only the `::class` usage is left.

**Review by MrPetovan — APPROVED** [review]
Thank you for your work!

**Pull request merged**